Installed/Exported FSeam CMake-Package seems faulty

**Describe the bug**
When I build FSeam with CMake and install the FSeam package afterwards, I run into various problems when I try to use it with the `FSeam_tutorial` project:

* PROBLEM 1: During cmake build-dir creation: 
  Headers seem to be expected under `INSTALL_ROOT/include` and not under `INSTALL_ROOT/share/include`
* PROBLEM 2: While building with cmake/ninja: Late discovery that `FILE_FSEAMER_PY-NOTFOUND` (with python3.8)

HINT: FSeam is installed into an empty INSTALL_ROOT directory.

**To Reproduce**
Directory tree:

```
+-- .installed/       -- Empty INSTALL_ROOT directory
+-- FSeam/            -- Cloned git repository
+-- FSeam_tutorial/   -- Cloned git repository
```
Steps to reproduce the behavior:

```sh
# -- FSeam project: Install FSeam into INSTALL_ROOT=.installed
$ cd FSeam/
$ mkdir build && cd build 
$ cmake -D CMAKE_INSTALL_PREFIX=$PWD/../../.installed ..
$ cmake --install .
```

```sh
# -- FSeam_tutorial project: 
$ cd FSeam_tutorial/
$ mkdir build && cd build 
$ FSeam_DIR=$PWD/../../.installed cmake ..
# -- EXPECT: PROBLEM 1 => .installed/include does not exist
# -- EXPECT: PROBLEM 2 => Found Python3 ... FILE_FSEAMER_PY-NOTFOUND
```
HINT: PROBLEM 2 seems to occur if INSTALL_ROOT/bin is not in the search-path (PATH environment variable).

PROBLEM 1:
```sh
CMake Error in CMakeLists.txt:
  Imported target "FSeam::FSeam" includes non-existent path

    "/.../.installed/include"

  in its INTERFACE_INCLUDE_DIRECTORIES.  Possible reasons include:
 ...
```

**Expected behavior**
After installation of the FSeam cmake package, it should be usable by `FSeam_tutorial` project (or any other project that uses FSeam).

**Desktop (please complete the following information):**
 - OS: macOS
- cmake: 3.17.1, 3.18.0

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -41,7 +41,7 @@ include(CMakePackageConfigHelpers)
 install(TARGETS FSeam FSeam-static
         EXPORT ${PROJECT_NAME}-targets)
 
-install(FILES ${FSEAM_HEADERS} DESTINATION share/include/FSeam)
+install(FILES ${FSEAM_HEADERS} DESTINATION include/FSeam)
 install(PROGRAMS ${FSEAM_GENERATOR_PYTH} DESTINATION share/bin)
 install(PROGRAMS ${FSEAM_GENERATOR_PYTH} DESTINATION bin)
 

